core/arch/x86/decode_fast.c

```diff
@@ -472,7 +472,7 @@ static const byte xop_a_extra[256] = {
  */
 int
 decode_sizeof(dcontext_t *dcontext, byte *start_pc,
-              int *num_prefixes _IF_X64(uint *rip_rel_pos))
+              int *num_prefixes _IF_X86_64(uint *rip_rel_pos))
```

Comment:
This is a nop: prob best for history to not change the line

Reply:
Reverted.